Assets/ModelAndAnim/FizzySD/DK Compagnon.otf.meta

```diff
@@ -0,0 +1,21 @@
+fileFormatVersion: 2
```

Comment:
Based on the license of this font: https://www.1001freefonts.com/view-pdf/20019/SGFub2RlZCBGb250cyBMaWNlbnNlICYgRkFRIC0gRE8gUkVBRCBUSElTIS5wZGY=/hanoded-fonts-license---faq---do-read-this-.pdf , we don't use it. So delete it